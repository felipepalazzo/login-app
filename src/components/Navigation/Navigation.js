import React from 'react'
import './Navigation.scss'
import flag from '../../assets/img/flag-dark.svg'
import arrow from '../../assets/img/arrow-down-dark.svg'

export default function Navigation() {
	return (
		<header className="header">
			<nav className="navbar container">
				<div className="navbar__content col-xs-12">
					<a href="https://monese.com/">Monese</a>
					<div type="navbar__option">
						<img src={flag} alt="" />
						<span className="navbar__option__text">English</span>
						<img src={arrow} alt="" />
					</div>
				</div>
			</nav>
		</header>
	)
}
