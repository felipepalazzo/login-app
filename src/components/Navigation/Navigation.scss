@import 'src/styles/colors';

.header {
	background-color: $white;
	position: fixed;
	height: 80px;
	width: 100%;
}

.navbar,
.navbar__content {
	height: 100%;
}

.navbar__content {
	display: flex;
	justify-content: space-between;
	align-items: center;
}

.navbar__option__text {
	display: inline-block;
	font-family: 'Mark Offc Pro';
	font-size: 14px;
	font-weight: normal;
	line-height: 1.71;
	letter-spacing: normal;
	color: $dark-indigo;
	padding: 0 10px;
}
