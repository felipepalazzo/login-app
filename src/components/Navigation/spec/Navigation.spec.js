import React from 'react'
import { shallow } from 'enzyme'
import Navigation from '../Navigation'

describe('Navigation', () => {
	it('renders without crashing', () => {
		const wrapper = shallow(<Navigation />)
		expect(wrapper).toMatchSnapshot()
	})
})
