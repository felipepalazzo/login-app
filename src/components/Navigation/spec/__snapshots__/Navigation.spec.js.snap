// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navigation renders without crashing 1`] = `ShallowWrapper {}`;
